docs/content/glossary/glossary.mdx

```diff
@@ -0,0 +1,318 @@
+---
+title: Glossary
+---
+
+# Flow Glossary
+
+## All words and definitions related to Blockchain and Flow
+
+## **A**
+
+## Authentication
+[Authentication](https://docs.onflow.org/fcl/reference/authentication/#gatsby-focus-wrapper) connects users to their wallets, 
```

Comment:
let's always remove the `#gatsby...` - this is a leftover from a search

Reply:
fixed.